extras/kms/migrations/postgres/05_key_rewrap.up.sql

```diff
@@ -0,0 +1,24 @@
+begin;
+
+
+-- we need to make the key and version columns mutable in order to support
+-- rewrapping them root key versions.
```

Comment:
I like the colloquial "rewrapping them root key versions" but I am guessing some word or so is actually missing :-)

Reply:
Or maybe you just meant "the"

Reply:
ty